firefox-ios/Client/Application/AppLaunchUtil.swift

```diff
@@ -78,7 +78,13 @@ class AppLaunchUtil {
             }
         }
 
-        RustFirefoxAccounts.startup(prefs: profile.prefs) { _ in
+        // RustFirefoxAccounts doesn't have access to the feature flags
+        // So we check the nimbus flag here before sending it to the startup
+        var fxafeatures: RustFxAFeatures = []
+        if LegacyFeatureFlagsManager.shared.isFeatureEnabled(.closeRemoteTabs, checking: .buildAndUser) {
+            fxafeatures.insert(.closeRemoteTabs)
+        }
+        RustFirefoxAccounts.startup(prefs: profile.prefs, features: fxafeatures) { _ in
```

Comment:
It looks like we might need to teach `ShareViewController.setupUI()` about this, too—but let's save that for a follow-up!